study-builder/studies/brain/consent.conf

```diff
@@ -27,7 +27,9 @@
   "translatedSubtitles": [
     {
       "language": "en",
-      "text": "<div><span>If you have questions about the study or the consent form at any time, please contact us at </span><a href=\"tel:651-229-3480\" class=\"HeaderLink\">651-229-3480</a> or <a href='mailto:info@braincancerproject.org' class='HeaderLink'> info@braincancerproject.org</a>.</div>"
+      "text": """<p class="no-margin sticky__text"><span>If you have questions about the study or the consent form at any time, please contact us at </span>
```

Comment:
consent is  existing activity. We cannot deploy the activity again. It has to be  a patch to update existing activity template text.

Reply:
Added patch code to look at updated consent.conf and update study config with styling changes